addons/mass_mailing/models/mail_notification.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class MailNotification(models.Model):
```

Comment:
So this is the part where mail.notification can become a "trace" right?

Maybe you could explain that a little bit in the class docstring if you can.